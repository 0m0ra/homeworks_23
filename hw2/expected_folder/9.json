{"error": "Input file does not have the correct extension."}