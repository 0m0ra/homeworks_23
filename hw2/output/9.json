{
   "status": "FileNotFoundError",
   "message": "Is not a file path"
}