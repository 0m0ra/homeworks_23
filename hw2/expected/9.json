{
   "0-18": 0.0,
   "18-25": 0.0,
   "26-45": 0.0,
   "45-59": 0.0,
   "60+": 0.0
}